diagrams/sequence-diagrams/README.md

```diff
@@ -29,15 +30,22 @@ graph TD
         runner(forms-runner)
     end
 
+    creator---runner
+
```

Comment:
At some point the creator (or someone else from their org) needs to visit Pay to generate the payment link and put it into admin - should we capture that here?
```suggestion
creator -.- pay
```